ca/ca.go

```diff
@@ -123,7 +125,15 @@ func NewCAMetrics(stats prometheus.Registerer) *caMetrics {
 		})
 	stats.MustRegister(lintErrorCount)
 
-	return &caMetrics{signatureCount, signErrorCount, lintErrorCount}
+	certificates := prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "certificates",
+			Help: "Number of certificates issued",
```

Comment:
```suggestion
			Help: "Number of certificates issued, tagged by profile name",
```

Reply:
I've noticed some of our other help texts recently have the "labeled by foo" text - why is that? It only serves to make the help text longer, and get out of sync with the actual labels. Any given timeseries from this metric will always show the labels, and our tooling also makes it easy to see the labels for a given metric name (easier, in fact, than the help text). My preference is to not include "labeled by foo" in our metric help text.

Reply:
Fair enough! Maybe I'm holding it wrong, but I feel like I've run into multiple scenarios where grafana wasn't surfacing the labels I could filter by. You're absolutely right that it doesn't do a good job of surfacing the help text either, though.